packages/terra-clinical-detail-view/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ Unreleased
 ### Changed
 * Update wdio specs to use Terra.describeViewports
 
+### Fixed
+* Fixed IE10 wrapping issue for label-value-views inside of detail-view-list.
```

Comment:
want to update this entry? probably state we added a max width to the item that resolves wrapping bug in ie 10

Reply:
Updated the entry in CHANGELOG.md file to be more specific.